Agar.io/Assets/Scripts/Model/AgarioGame.cs

```diff
@@ -24,12 +26,13 @@ private void SpawnFood()
 
             for (var i = 0; i < FoodCount; i++)
             {
-                int x = random.Next(-3,3);
-                int y = random.Next(-3,3);
+                int x = random.Next(0, 100);
+                int y = random.Next(0, 100);
```

Comment:
Why range is from 0 to 100? It will spawn food only in upper rigth corner of the map, isn't it?

Reply:
It will be easier to work with chunks if our board will be a square from 0 to 100 on both x and y axes